pkg/sensors/tracing/kprobe_process_credentials_test.go

```diff
@@ -180,6 +181,11 @@ spec:
 		t.Fatalf("setgid(%d) error: %s", gid+1, err)
 	}
 
+	// Restore all gids to 0
+	if err := syscall.Setgid(0); err != nil {
+		t.Fatalf("setgid(0) error: %s", err)
+	}
```

Comment:
Is this a fix for an existing test? Please have it as a separate patch.

Reply:
ok thanks